### PR TITLE
RPC Developer reference updated based on 0.12.1

### PR DESCRIPTION
**Pull request by Mirobit**
I updated all changes to the RPC API from 0.11.0 to 0.12.1.
An update based on 0.13.1 will follow soon.

### CODE REVIEW

**Comment by harding** [comment]
Wow, this is awesome!  Thanks, @Mirobit!  I haven't generated a local preview, but I took a quick skim at the diff and it looks good.  I'll try to make some time today or tomorrow to preview each of the changes.

**Comment by jonasschnelli** [comment]
Thanks for doing this!
An update to 0.13.1 would be highly appreciated.

File: _includes/devdoc/bitcoin-core/rpcs/rpcs/listtransactions.md [hunk 1]
```diff
@@ -188,7 +203,9 @@ Result:
         "walletconflicts" : [
         ],
         "time" : 1418695703,
-        "timereceived" : 1418925580
+        "timereceived" : 1418925580,
```
Comment (harding) [inline _includes/devdoc/bitcoin-core/rpcs/rpcs/listtransactions.md]: This example doesn't include the new `trusted` field documented above.

Comment (Mirobit) [inline _includes/devdoc/bitcoin-core/rpcs/rpcs/listtransactions.md]: The trusted field is only shown when the transaction is still unconfirmed. I didn't have an unconfirmed tx in my wallet.

**Comment by Mirobit** [comment]
Do you want me to squash all commits?

**Comment by harding** [comment]
@Mirobit Only if you want to squash them.  We've never been strict squashers on Bitcoin.org.

Unless someone discovers an issue, I suspect that this will be merge the next time @Cobra-Bitcoin processes the PR queue.

**Pull request merged**